pyinfra/operations/files.py

```diff
@@ -778,6 +778,56 @@ def get(
             yield FileDownloadCommand(src, dest, remote_temp_filename=host.get_temp_filename(dest))
 
 
+def _canonicalize_timespec(field, local_file, timespec):
+    assert field == "atime" or field == "mtime"
+    if isinstance(timespec, datetime):
+        if not timespec.tzinfo:
+            # specify remote host timezone
+            timespec_with_tz = timespec.replace(tzinfo=host.get_fact(Date).tzinfo)
+            return timespec_with_tz
+        else:
+            return timespec
+    elif isinstance(timespec, bool) and timespec:
+        lf_stat = os.stat(local_file)
+        if field == "atime":
+            return datetime.fromtimestamp(lf_stat.st_atime, tz=timezone.utc)
+        else:
+            return datetime.fromtimestamp(lf_stat.st_mtime, tz=timezone.utc)
```

Comment:
```suggestion
        tstamp = if_stat.st_atime if field == "atime" else if_stat.st_mtime
        return datetime.fromtimestamp(tstamp, tz=timezone.utc)
```

Nitpicking, just a bit neater.